sphinx/templates/quickstart/conf.py_t

```diff
@@ -141,7 +141,13 @@ latex_documents = [
     (master_doc, '{{ project_fn }}.tex', u'{{ project_doc_texescaped_str }}',
      u'{{ author_texescaped_str }}', 'manual'),
 ]
-
+{% if language|default('').split('_')[0] == 'zh' %}
```

Comment:
this runs correct in my environment, but I haven't touched tox before and I can not configure it to work correctly for now
I run this program successfully, on windows anaconda python3